silkworm/rpc/http/channel_writer.hpp

```diff
@@ -0,0 +1,56 @@
+/*
+   Copyright 2023 The Silkworm Authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+#pragma once
+
+namespace silkworm::rpc::http {
```

Comment:
If I understand correctly the purpose of defining this new interface `ChannelWriter`, it should be placed in `silkworm::rpc` namespace (it is not meant to be HTTP-specific, in fact the opposite is true).